### DISCUSSION TITLE
List view improvements in Nemo.

### DISCUSSION THREAD

**Discussion by iguanajuice** [linuxmint/discussions#208] [Ideas]
Hey all! I've just started using list view in Nemo and I quite like it. But there are two things that peeve me:

Quickly clicking on that that tiny arrow is pretty hard, and it's made even more difficult because the sidebar tries to "grab" the cursor for resizing. I suggest making the icon part of the arrow's click target:
![image](https://github.com/linuxmint/discussions/assets/125163000/e105278b-1dff-40ff-a96f-3a4f115a7594)

Secondly, the font size being inconsistent when changing the view scale. When using 100% view scaling in list mode, the text is comically large.  Thunar doesn't behave this way, and it looks a lot better because of it (imho).

Comparison between Thunar (left) and Nemo (right):
![image](https://github.com/linuxmint/discussions/assets/125163000/05b8cb04-3747-4709-b7e9-fcdece0d2a04)

**Comment by anaximeno** [comment]
Instead of making the icon part of the click target I believe it would be better to put a layer or something above or around the arrow and make it clickable so that all of the section where the arrow is would be clickable, and also putting a bit more margin between the arrow and the sidebar line.

**Comment by rcalixte** [comment]
I have different arrows in my file browser so I wonder if some of this is controlled or customizable by the theme. In general, I also tend to use Shift + Left/Right to collapse/expand in the list view. I didn't have any issues clicking the arrows though, possibly because the arrows in my theme are different.

**Reply by iguanajuice** [reply]
I have tried 3 different themes (Adwaita, Adapta (shown above), and Materia) and they all have the same size arrow. Would you care to show me screenshot of your Nemo's list view and the name of the theme you are using? Thanks.

**Reply by rcalixte** [reply]
This is Mint-Y-Dark-Grey with the zoom setting I use (I think it's one below the regular):

![Screenshot from 2023-11-15 13-17-33](https://github.com/linuxmint/discussions/assets/10281587/0931b360-62fa-475a-9e50-624d5f9e96b9)


Maybe it's because of the zoom setting that I'm using that it looks fine? (I just checked and mine is set at a default zoom for the list view at 66%.)

**Reply by iguanajuice** [reply]
The arrows appear to be the same size as mine? As in, too small to consistently click on (without grabbing the side pane border for resizing).

**Reply by rcalixte** [reply]
I've never had any issues. My cursor fits exactly over the icon too. I tend to use the keyboard more often if I'm browsing directories as well. Maybe a setting for the arrow size would be a good compromise?

**Reply by JosephMcc** [reply]
I have to admit I've never had this issue either. The arrows are controlled by the combo of the gtk theme and the icon theme. The sidebar grabbing the cursor could also be more of a problem in certain themes. The arrows should also highlight when hovered so that you know that you are over them.

**Reply by iguanajuice** [reply]
I understand that the size of the arrow is mainly dictated by the current theme, but I'm not asking to make the arrows bigger; I'm suggesting making the icon apart of the arrows click target.

Currently, clicking the icon doesn't do anything special, it just selects (or opens, if using single-click) the target file/directory.
